tests/ui.rs

```diff
@@ -145,23 +248,56 @@ fn run_tests(
     with_dependencies: bool,
     tmpdir: &Path,
 ) -> Result<()> {
+    // Handle command-line arguments.
+    let mut args = ui_test::Args::test()?;
+    args.bless |= env::var_os("RUSTC_BLESS").is_some_and(|v| v != "0");
+
+    let with_dependencies = with_dependencies.then_some(WithDependencies { bless: args.bless });
+
     let mut config = miri_config(target, path, mode, with_dependencies);
+    config.with_args(&args);
+    if let Mode::Pass { rustc: true } = mode {
+        config.fill_host_and_target()?;
+        // Rustc mode only works on the host
+        if !config.host_matches_target() {
+            return Ok(());
+        }
+        config.output_conflict_handling = ui_test::ignore_output_conflict;
```

Comment:
Why do we want to ignore output conflicts when building with rustc?

Reply:
This is compilation output. We never have compilation output from miri, because that normally means the test can't run at all. I have not seen a single example in the run test suites that show warnings except for miri specific warnings

Reply:
I could force the output to be empty